cmd/flags/service.go

```diff
@@ -49,18 +49,23 @@ type Service struct {
 	MetricsFactory metrics.Factory
 
 	signalsChannel chan os.Signal
+
+	// HcStatusChannel is used for error propagation
+	HcStatusChannel chan healthcheck.Status
```

Comment:
this doesn't need to be exposed publicly, instead you can have a method SetHealthCheckStatus() - assuming this is how we want goroutines to interact  with Service